Tests/DangerTests/GitHubTestResources/GitHubUser.swift

```diff
@@ -0,0 +1,6 @@
+
+public let GitHubUserJSON = """
```

Comment:
I'm not against this but honestly just running the tests with `swift test` would suffice 🤔

Reply:
I think I misinterpreted what you were saying before. Essentially, the test bundle is just a requirement by xcode/xcodebuild, but with `swift test` the json file could be easily found without using a build script or makefile?

Reply:
It's fine I likely confused you because I know so little about what I'm talking about 😂 I think the root directory when using `swift test` is the location of the package manifest?

Reply:
I'll look more into this on Monday or Tuesday!